api/src/main/java/org/openmrs/api/impl/OrderServiceImpl.java

```diff
@@ -13,6 +13,12 @@
 import static org.openmrs.Order.Action.REVISE;
 
 import java.lang.reflect.Field;
+
+import org.apache.commons.lang.time.DateUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.hibernate.proxy.HibernateProxy;
```

Comment:
Did you configure your IDE to use the OpenMRS code formatter? org.* should be coming after java.*